code/world.dm

```diff
@@ -35,6 +35,9 @@ var/datum/early_init/early_init_datum = new
 
 /world/New()
 	world_startup_time = world.timeofday
+	#if DM_BUILD >= 1506
```

Comment:
There's really no need for this. The code won't compile on anything nearly as old as 1506.